src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pyx

```diff
@@ -74,7 +68,41 @@ cdef class CompletionQueue:
     with self.poll_condition:
       self.is_polling = False
       self.poll_condition.notify()
+    return self.interpret_event(event)
+
+  def pluck(self, records.OperationTag tag, records.Timespec deadline=None):
+    # Plucking a 'None' tag is equivalent to passing control to GRPC core until
+    # the deadline.
+    # TODO(atash) there's currently no way of getting a tag to pass to this
```

Comment:
If the code above this were changed to take a break from continuously calling `next` and call this `pluck` function with a tag that it knew was outstanding, would this method misbehave?

If no value other than `None` is safe to pass for `tag`, why even have the parameter?